From a85270d07e7a249151e6b59b62d68179a818ca39 Mon Sep 17 00:00:00 2001
From: Tomasz Figa <tfiga@chromium.org>
Date: Fri, 3 Jun 2016 16:06:40 +0900
Subject: [PATCH 514/517] CHROMIUM: i965: Advertise R8G8B8A8 and R8G8B8X8 DRI
 configs

Both formats are supported by hardware and some clients, such as
Android, require them.

Signed-off-by: Tomasz Figa <tfiga@chromium.org>
(cherry picked from commit 3109f33214bb07f5bcc69f24833cbb8d42ac511f)

BUG=b:29036398
TEST=No significant regressions in Android dEQP

Change-Id: I987069f945cf2bf370724d4116e08e97bf76e488
---
 src/mesa/drivers/dri/i965/intel_screen.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/mesa/drivers/dri/i965/intel_screen.c b/src/mesa/drivers/dri/i965/intel_screen.c
index fec7b47..3198419 100644
--- a/src/mesa/drivers/dri/i965/intel_screen.c
+++ b/src/mesa/drivers/dri/i965/intel_screen.c
@@ -1273,6 +1273,8 @@ intel_screen_make_configs(__DRIscreen *dri_screen)
 {
    static const mesa_format formats[] = {
       MESA_FORMAT_B5G6R5_UNORM,
+      MESA_FORMAT_R8G8B8A8_UNORM,
+      MESA_FORMAT_R8G8B8X8_UNORM,
       MESA_FORMAT_B8G8R8A8_UNORM,
       MESA_FORMAT_B8G8R8X8_UNORM
    };
-- 
2.8.0.rc3.226.g39d4020

